src/Symfony/Bridge/Twig/composer.json

```diff
@@ -22,6 +22,7 @@
     "require-dev": {
         "fig/link-util": "^1.0",
         "symfony/asset": "~2.8|~3.0|~4.0",
+        "symfony/dependency-injection": "~2.8|~3.0|~4.0",
```

Comment:
this targets 3.4, should this be `~3.4|~4.0`?

Reply:
I just need the `ContainerInterface` for the tests. So I guess `2.8` is also fine? Not sure what to do here tbh :blush: